frontend/X-Change/OpenOrders/OpenOrdersDiode.qml

```diff
@@ -0,0 +1,18 @@
+import QtQuick 2.7
+import QtQuick.Layouts 1.3
+
+import "../../Controls" as Controls
+
+Controls.Diode {
+    id: openOrders
+    width: parent.width - 100
+    height: parent.height - 500
+    Layout.minimumHeight: 100
+    color: "#3A3E47"

```

Comment:
The default background diode colour is set by Controls/Diode to Theme.panelBackground which is this value. Colour refs can be removed (making it easy to tweak them all later when fine-tuning design)